collector/collector.lua

```diff
@@ -59,8 +61,12 @@ function curate(set)
 			data = inventory[bag][i]
 			if data.id ~= 0 then
                 local name = res_items[data.id].name
+                local level = res_items[data.id].item_level or res_items[data.id].level or ''
+                local ag = item_descriptions[data.id] and item_descriptions[data.id].en:find("Afterglow") and true or false
+                level = level == 99 and ag and (level .. " II") or level == 119 and ag and (level .. " III") or level
```

Comment:
The bottom two lines needs to go under an `if` that checks if it's actually needed (i.e. `level and level == 99`, most likely). The least we touch `item_description` table the better, and due to how resources work it's not actually loaded until it's accessed the very first time.

Reply:
Fixed in [a84395a](https://github.com/lili-ffxi/FFXI-Addons/pull/13/commits/a84395a2c2d509121138be33ee702a5be90401bc)